pwiz_tools/Skyline/Model/TransitionGroupDocNode.cs

```diff
@@ -247,6 +248,12 @@ public IList<DocNode> GetPrecursorChoices(SrmSettings settings, ExplicitMods mod
 
         public SpectrumClassFilter SpectrumClassFilter { get; private set; }

```

Comment:
SpectrumClassFilter here but SpectrumFilter in TransitionSettingsUI.

Reply:
So I'm still curious what is the reason for the "Class" bit in the name?

Reply:
> So I'm still curious what is the reason for the "Class" bit in the name?

Skyline already had a class called "SpectrumFilter" which is a pretty important class which does all of the work of extracting chromatograms from spectra. In that class name, the word "filter" basically means "transformer".
The feature that I implemented last year is called "Spectrum Filter" in the user interface, but it had to be called something else in the code because the name "SpectrumFilter" was already being used.

Reply:
Did you consider "SpectrumMetadataFilter"? Not a big deal obviously, just something to ponder.